packages/itwinui-css/src/table/base.scss

```diff
@@ -154,7 +153,8 @@
   display: flex;
 
   &:last-of-type {
-    border-radius: inherit;
+    border-end-start-radius: var(--iui-border-radius-1);
+    border-end-end-radius: var(--iui-border-radius-1);
```

Comment:
This is also getting applied to the table-header row, which is not desirable.

I believe we only want it for rows inside the table-body, and only when there is no paginator.

Reply:
Also there might be some remaining instances of `border-radius: inherit` that could be removed.

Reply:
Updated the selectors so that the border radius is only applied in table-body when there is not paginator in this [commit](https://github.com/iTwin/iTwinUI/pull/2143/commits/452e3fe26f0daf7e83d6e5dad3d41f8a141a948f).

Trying to remove the last instance of `border-radius: inherit` from the file as well, and currently causing unexpected visual changes. The instance is on the first table cell of a row, currently trying to create a selector so that the border radius appears when we expect it to.

Reply:
The last instance of `border-radius: inherit` has been correctly replaced in this [commit](https://github.com/iTwin/iTwinUI/pull/2143/commits/9089064cb8da806af525cef40b967c10bc36b16e). Also updated a react screenshot to remove a curved bottom left corner from the header row.

Reply:
Could you explain this long selector? You lost me here 😅

![](https://github.com/user-attachments/assets/11162c04-e82a-406c-82a2-694cd618216b)

Also there is another instance of `border-radius` left in the React code.

https://github.com/iTwin/iTwinUI/blob/e4b624be6e9e3baf08962c99a1c3c42a22400401/packages/itwinui-react/src/core/Table/Table.tsx#L1156-L1161

Reply:
Okay, so after selecting `.iui-table-body`s where there is no `.iui-table-paginator` present, we select the last `.iui-table-row` child, and select the first `.iui-table-cell` child of the row to add the bottom left border radius. Added a comment to explain what the selector is doing in this [commit](https://github.com/iTwin/iTwinUI/pull/2143/commits/2cb214c19ad85e29a973a0e52c3d2583f1eb1c80).

Also removed the inherited `border-radius` from the React code in this [commit](https://github.com/iTwin/iTwinUI/pull/2143/commits/9d99d206a8b5d48ab5560a0617ab8d00ad79218b)